ci/docs/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright (c) 2021, NVIDIA CORPORATION.
+######################################
+# Dask-CUDA Docs build script for CI #
+######################################
+
+if [ -z "$PROJECT_WORKSPACE" ]; then
+    echo ">>>> ERROR: Could not detect PROJECT_WORKSPACE in environment"
+    echo ">>>> WARNING: This script contains git commands meant for automated building, do not run locally"
+    exit 1
+fi
+
+export DOCS_WORKSPACE=$WORKSPACE/docs
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
+export HOME=$WORKSPACE
+export PROJECT_WORKSPACE=/rapids/dask-cuda
+export NIGHTLY_VERSION=$(echo $BRANCH_VERSION | awk -F. '{print $2}')
```

Comment:
`HOME` and `NIGHTLY_VERSION` are exported but unused (copied this from cuDF's build script); are these required by gpuCI or can I remove these?

Reply:
HOME is a variable needed by Jenkins, and NIGHTLY_VERSION is used in the build outside this script.